spec/spec_tests/data/connection_string/invalid-uris.yml

```diff
@@ -7,14 +7,6 @@ tests:
         hosts: ~
         auth: ~
         options: ~
-    -
-        description: "Invalid scheme"
```

Comment:
this should be valid under the new SRV requirements, right?

Reply:
The new SRV requirements only apply to `mongodb+srv` schemes. Note the scheme for this test is `mongo`, which is not a supported scheme. (We only support `mongodb://` and `mongodb+srv://` -- which is what this test is ensuring.)

So, this test should remain unchanged -- and it should be passing. If it is failing with your changes, then you might have touched more than you intended...

Reply:
fixed! thanks!